h11/_headers.py

```diff
@@ -77,10 +77,13 @@ def normalize_and_validate(headers, _parsed=False):
             validate(_field_value_re, value, "Illegal header value {!r}", value)
         name = name.lower()
         if name == b"content-length":
-            if saw_content_length:
-                raise LocalProtocolError("multiple Content-Length headers")
             validate(_content_length_re, value, "bad Content-Length")
-            saw_content_length = True
+            if saw_content_length:
```

Comment:
We'd also like to handle `Content-Length: x, x, x, ...` -> `Content-Length: x`. cc @njsmith

Reply:
Should it also be `if saw_content_length is not None`, say if one header is `Content-Length: 0`?